cleanlab/multilabel_classification/rank.py

```diff
@@ -110,14 +139,96 @@ def get_label_quality_scores(
     >>> scores
     array([0.9, 0.5])
     """
-
-    assert_valid_inputs(
-        X=None, y=labels, pred_probs=pred_probs, multi_label=True, allow_one_class=True
+    binary_labels = _labels_to_binary(labels, pred_probs)
+    scorer, base_scorer_kwargs = _create_multilabel_scorer(
+        method=method,
+        adjust_pred_probs=adjust_pred_probs,
+        aggregator_kwargs=aggregator_kwargs,
     )
-    num_classes = get_num_classes(labels=labels, pred_probs=pred_probs, multi_label=True)
-    binary_labels = int2onehot(labels, K=num_classes)
-    base_scorer = ClassLabelScorer.from_str(method)
-    base_scorer_kwargs = {"adjust_pred_probs": adjust_pred_probs}
-    aggregator = Aggregator(**aggregator_kwargs)
-    scorer = MultilabelScorer(base_scorer, aggregator)
     return scorer(binary_labels, pred_probs, base_scorer_kwargs=base_scorer_kwargs)
+
+
+def get_class_label_quality_scores(
+    labels: List[List[int]],
+    pred_probs: npt.NDArray["np.floating[T]"],
+    *,
+    method: str = "self_confidence",
+    adjust_pred_probs: bool = False,
+) -> np.ndarray:
+    """Computes a per class label quality score for each example in a multi-label classification dataset.
+
+    Scores are between 0 and 1 with lower scores indicating examples whose label more likely contains an error.
+    For each example, this method internally computes a separate score for each individual class
+    and then aggregates these per-class scores into an overall label quality score for the example.
+
+    To estimate exactly which examples are mislabeled in a multi-label classification dataset,
+    you can also use :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` with argument ``multi_label=True``.
```

Comment:
be careful with copy/paste! This seems very unnecessary to mention for the per-class version of label-quality-scoring. You should've instead linked to `cleanlab.multilabel_classification.filter.find_multilabel_issues_per_class` which is ofc the more analogous method to this one.

Reply:
Re-wrote docs: https://github.com/cleanlab/cleanlab/blob/65432bbebb1d0767fb49ae988920a2250cde3e24/cleanlab/multilabel_classification/rank.py#L75